src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const titleToLower = movie.title.toLowerCase();
+    const descriptionToLower = movie.description.toLowerCase();
+    const queryToLower = query.toLowerCase();
```

Comment:
Great job with creating variables for making the code clearer 👍 
You can make it better still. Create boolean conditions for `doesTitleMatch / doesDescriptionMatch` instead of those